jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-client/src/main/java/org/jbpm/console/ng/cm/client/comments/CaseCommentItemView.java

```diff
@@ -173,4 +201,79 @@ public void updateCommentPressingEnter(@ForEvent("keyup") final KeyboardEvent e)
         }
     }
 
+    @Override
+    public CaseCommentSummary getValue() {
+        return caseCommentSummary.getModel();
+    }
+
+    @Override
+    public void setValue(final CaseCommentSummary model) {
+        this.caseCommentSummary.setModel(model);
+        this.updateCommentText.setValue(model.getText());
+        setEditMode(editMode);
+    }
+
+    protected void updateActions(final String author,final String id, final boolean editItem){
+        actionsItems.setInnerHTML("");
+        if (identity.getIdentifier().equals(author)) {
+            addAction( new CaseCommentsPresenter.CaseCommentAction() {
+                @Override
+                public String label() {
+                    return translationService.format(DELETE);
+                }
+
+                @Override
+                public void execute() {
+                    presenter.deleteCaseComment(id);
+                }
+            });
+
+            if (editItem) {
+                addUpdateCommentAction(new CaseCommentsPresenter.CaseCommentAction() {
+                    @Override
+                    public String label() {
+                        return translationService.format(SAVE);
+                    }
+
+                    @Override
+                    public void execute() {
+                        if (validateForm()) {
+                            presenter.updateCaseComment(getUpdatedComment(),id);
```

Comment:
Same here, pass the CaseCommentSummary object, which should already contain the new value for the comment in the text attribute, set via `@Bound`.